packages/web3-shh/types/index.d.ts

```diff
@@ -0,0 +1,130 @@
+/*
+    This file is part of web3.js.
+    web3.js is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    web3.js is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+    You should have received a copy of the GNU Lesser General Public License
+    along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/**
+ * @file index.d.ts
+ * @author Josh Stevens <joshstevens19@hotmail.co.uk>, Samuel Furter <samuel@ethereum.org>
+ * @date 2018
+ */
+
+import * as net from 'net';
+import { AbstractWeb3Module, provider, Web3ModuleOptions } from 'web3-core';
+import { Network } from 'web3-net';
+
+export class Shh extends AbstractWeb3Module {
+    constructor(
+        provider: provider,
+        net?: net.Socket|null,
+        options?: Web3ModuleOptions
```

Comment:
The constructor parameters aren't aligned with the 1.0 interface. (options can't get passed)

Reply:
it looks like in the function in index.js nothing gets passed to it at all.. so will remove all.

```function Shh() {```